geth/node/node.go

```diff
@@ -166,12 +142,7 @@ func activateEthService(stack *node.Node, config *params.NodeConfig) error {
 	ethConf.DatabaseCache = config.LightEthConfig.DatabaseCache
 
 	if err := stack.Register(func(ctx *node.ServiceContext) (node.Service, error) {
-		lightEth, err := les.New(ctx, &ethConf)
-		if err == nil {
-			updateCHT(lightEth, config)
```

Comment:
This is pretty important for us as otherwise LES must be synced from the beginning. How is it handled now? LES/2 feature?

Reply:
I have just read it in the comment... 

One way to test it is to run a full node with LES enabled and try to sync your local node using that full node as a peer. We have a full node in our cluster. Let me know if you don't have access.